src/extends/Group/index.tsx

```diff
@@ -18,6 +18,10 @@ type Props = {
   field?: Field;
   jsonKey?: string;
   propertyValue?: any;
+  alwaysShow?: boolean;
+  disableAddon?: boolean;
+  titleLeftGrid?: string;
```

Comment:
Why add titleLeftGrid and titleRightGrid?

Reply:
Remove it .done